crtclient/crtclient.go

```diff
@@ -15,29 +15,26 @@ limitations under the License.
 
 package crtclient
 
-// container runtime client interface implementations
-
-type Config struct {
-	// Config object parsed from a json styled config
-	V interface{}
-}
-
-type ContainerEpContext struct {
+// ContainerEPContext is used to manage the parameters for all
+// container+endpoint management operations.
+type ContainerEPContext struct {
 	NewContName    string
 	NewAttachUUID  string
 	CurrContName   string
 	CurrAttachUUID string
-	InterfaceId    string
-	IpAddress      string
+	InterfaceID    string
+	IPAddress      string
 	SubnetLen      uint
 	DefaultGw      string
 }
 
+// ContainerIf implementations are used to configure and manage container
+// interfaces.
 type ContainerIf interface {
-	Init(config *Config) error
+	Init(config interface{}) error
```

Comment:
I've reverted this in this patch and intend to introduce it more globally in a future one.